### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by devonY3303**
# Description

Clean up code

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: Front end/components/game page/style.css [hunk 1]
```diff
@@ -226,6 +227,7 @@ section h1 {
     font-size: 3em;
     color: white;
 }
+
 @media (max-width: 600px) {
     .mobile-title{
         /* margin-top: 3%;
```
Comment (Lior-Becker) [inline Front end/components/game page/style.css:233]: missed a comment

File: Front end/components/game page/style.css [hunk 2]
```diff
@@ -244,13 +247,16 @@ section h1 {
...
         padding-top: 5%;
     }
+
     .game-right-side {
         display: none;
     }
+
     .mobile-title h1{
         font-size: 3em;
         /* text-wrap: nowrap; */
```
Comment (Lior-Becker) [inline Front end/components/game page/style.css:257]: missed another comment

File: Front end/components/game page/style.css [hunk 3]
```diff
@@ -244,13 +247,16 @@ section h1 {
...
     }
+
     .game-full-content{
         /* margin: 0%; */
```
Comment (Lior-Becker) [inline Front end/components/game page/style.css:261]: comment

**Review by Lior-Becker — APPROVED** [review]

**Review by jhviljoen — APPROVED** [review]
I'm Commander Shepard, and this is my favorite PR on the Citadel.

**Pull request merged**